test/0220-test-configs.t

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+. testlib
+plan_tests 15
+
+# Test that config files that we ship in test_config can be at least
+# parsed. Several of them are hardware dependent and cannot be
+# executed on arbitrary hardware. Therefore, their automated testing
+# is problematic.
+
+# set fixed log level for config tests; otherwise if user would run something
+#  like `SPDLOG_LEVEL=trace make test`, the output would include the logs and the tests would fail
+export SPDLOG_LEVEL=warning
+
+test_config() {
+    local cfg_in=$1
+
+    okx demos-sched -d -c "../test_config/$cfg_in"
+}
+
+test_config CPU_switching.yaml
```

Comment:
I don't like the explicit listing, I'd prefer just testing all configs in the directory, that way I don't have to remember to add them here.

Reply:
Yes, but I have that directory full of many temporary config files etc. and I don't want to see random failures caused by them.